src/components/badges/DiscordBadge.tsx

```diff
@@ -0,0 +1,22 @@
+/* @jsx MD */
+import MD, { Component } from "jsx-md";
+import { Badge } from "../Badge";
+
+/** Show a discord badge  on GitHub */
```

Comment:
This line needs to be directly in front of the DiscordBadge line (before line 13). And there is a double space in it.

Reply:
done